node/core/backing/src/lib.rs

```diff
@@ -593,23 +595,17 @@ async fn make_pov_available(
 ) -> Result<Result<(), InvalidErasureRoot>, Error> {
 	let available_data = AvailableData { pov, validation_data };
 
-	{
-		let _span = span.as_ref().map(|s| s.child("erasure-coding").with_candidate(candidate_hash));
-
-		let chunks = erasure_coding::obtain_chunks_v1(n_validators, &available_data)?;
-
-		let branches = erasure_coding::branches(chunks.as_ref());
-		let erasure_root = branches.root();
-
-		if erasure_root != expected_erasure_root {
-			return Ok(Err(InvalidErasureRoot))
-		}
-	}
-
 	{
 		let _span = span.as_ref().map(|s| s.child("store-data").with_candidate(candidate_hash));
 
-		store_available_data(sender, n_validators as u32, candidate_hash, available_data).await?;
+		store_available_data(
```

Comment:
Let's add a comment here to explain that this is checking the erasure root

Reply:
done